features/provide_vat_receipt.feature

```diff
@@ -0,0 +1,33 @@
+@US259
+Feature: Add VAT receipt at the bottom of the printed summary page
```

Comment:
I thought that the VAT receipt was meant to be separate. Is that no longer true?